generator/app.go

```diff
@@ -179,9 +179,15 @@ func writeLastGenerated(f *os.File) {
 }
 
 func createReadmeFiles(ctx SigEntries) error {
+	selectedSig := os.Getenv("SIG")
```

Comment:
nit: It is a little wacky to pass this through as an env variable vs. a command line flag. But it probably isn't worth changing at this point.